cli/src/commands/help.rs

```diff
@@ -26,21 +30,94 @@ pub(crate) struct HelpArgs {
     pub(crate) command: Vec<String>,
 }
 
+struct Category {
+    description: &'static str,
+    content: &'static str,
+}
+
 #[instrument(skip_all)]
 pub(crate) fn cmd_help(
-    _ui: &mut Ui,
+    ui: &mut Ui,
     command: &CommandHelper,
     args: &HelpArgs,
 ) -> Result<(), CommandError> {
+    // TODO: Add all documentation to categories
+    //
+    // Maybe adding some code to build.rs to find all the docs files and build the
+    // HashMap at compile time.
```

Comment:
nit: `HashMap` -> `BTreeMap`

btw, it might make sense to extract the inner array as file-local `const CATEGORIES: &[Category] = ..`, and add helper function that constructs `help_template_categories`.

Reply:
Is that what you meant?

Reply:
I meant something like:
```rust
const CATEGORIES: .. = &[
];

// will be embedded as #[command(after_long_help = help::format_categories_list())]
pub fn format_categories_list() -> String { // or clap's StyledStr?
    ...
}

// could be inilined
fn find_category(name) -> Option<&Category> {
    CATEGORIES.iter().find(..)
}
```